file/src/test/java/docs/javadsl/DirectoryTest.java

```diff
@@ -46,7 +45,7 @@
   @BeforeClass
   public static void beforeAll() throws Exception {
     system = ActorSystem.create();
-    materializer = ActorMaterializer.create(system);
+    materializer = Materializer.apply(system);
```

Comment:
We want to drop all the materializer references in the tests (where possible).

Reply:
@ennru sadly, there is a [method](https://github.com/akka/akka/blob/master/akka-stream-testkit/src/main/scala/akka/stream/testkit/javadsl/StreamTestKit.scala#L18) in the tearDown, requiring the Materializer instance. 

```java
  @After
  public void tearDown() throws Exception {
    fs.close();
    fs = null;
    StreamTestKit.assertAllStagesStopped(materializer);
  }
```

Reply:
You can get the materialized from an actor system, `Materializer.matFromSystem(system)`

Reply:
just pushed it changed